src/Components/App/App.js

```diff
@@ -14,13 +16,37 @@ class App extends Component {
       moviesAnswers:[],
       podcastsAnswers:[],
       boardGamesAnswers:[],
-      cardGamesAnswers:[]
+      cardGamesAnswers:[],
+      movies: [],
+      music: [],
+      podcasts: [],
+      boardGames: [],
+      cardGames: []
     }
   }
 
-  findSetOfQuestions = () => {
-  
-    // this.state
+  getActivityData = async (event) => {
+    let promise;
+    try{
+      if(event.target.id === 'movies' ) {
+          promise = await getAllMovies();
+      }
+      if(event.target.id === 'boardGames' ) {
+        promise = await getAllBoardGames();
+      }
+      if(event.target.id === 'cardGames' ) {
+        promise = await getAllCardGames();
+      }
+      if(event.target.id === 'music' ) {
+          promise = await getAllMusic();
+      }
+      if(event.target.id === 'podcasts' ) {
+        promise = await getAllPodcasts();
+      }
+      this.setState({ [event.target.id]: promise})
+      } catch (error){
+        console.log(error)
```

Comment:
Also, we will have to think about error handling and maybe setting an `error` in state to be rendered in `App` if these fail in some way.

Reply:
I think we can render something when an error happens or when the page is loading :)